src/js/messaging/actions/messages.js

```diff
@@ -8,17 +8,21 @@ import {
   DELETE_DRAFT_ATTACHMENT,
   DELETE_MESSAGE_FAILURE,
   DELETE_MESSAGE_SUCCESS,
+  DELETING_MESSAGE,
```

Comment:
Yes, I started adding them, and figured I should break the other loading-required actions into separate PRs.